src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -82,11 +52,12 @@ export const PreviouslyViewedLinks = () => {
     dispatch(closeTrackPanelModal());
   };
 
+  const limitedPreviouslyViewedObjects = previouslyViewedObjects.slice(0, 20);
```

Comment:
Is this constant even necessary?

Reply:
why? You mean using the slice directly on the map? I thought it is good for readability

Reply:
I'm slightly bothered by the word `limited` — to me, it makes the name of the variable more confusing than helpful. One other thought is that we could have done the slice back on line 36 (`useSelector(getActiveGenomePreviouslyViewedObjects).slice(0, 20)`). But this is just a stylistic remark; if you prefer the name `limitedPreviouslyViewedObjects`, feel free to ignore this comment.